lib/common-types.ts

```diff
@@ -566,6 +580,27 @@ export type MRActions = {
       return: AssetKeyProps
     }
   }
+  BulkAction: {
+    get: {
+      params: GetBulkActionParams
+      return: BulkActionProps
+    }
+    publish: {
+      params: GetSpaceEnvironmentParams
+      payload: any
+      return: BulkActionProps
+    }
+    unpublish: {
+      params: GetSpaceEnvironmentParams
+      payload: any
+      return: BulkActionProps
+    }
+    validate: {
+      params: GetSpaceEnvironmentParams
+      payload: any
+      return: BulkActionProps
```

Comment:
This should be `BulkActionProps<ValidateBulkActionPayload>`

Reply:
I like that 👌🏽